client/src/reducers/auth/passwordReducer.js

```diff
@@ -0,0 +1,74 @@
+import actionTypes from '../../actions/actionTypes';
+
+const initialState = {
+  message: null,
+  success: false,
+  errors: null,
+  fails: null,
+  reset_message: null,
+  reset_success: false,
+  reset_errors: null,
+  reset_fails: null
+};
+const passwordReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.RECOVERY_SUCCESSFUL:
```

Comment:
Redux state (reducers) are suppose to be pure functions. You cannot mutate state and so you can only return a copy of the state every time there is a change in the state. What I noticed is that you are mutating the state in all of your reducers and its not a good idea.

You might want to consider reviewing what you have already